### DISCUSSION TITLE
Disabling  ''Disable mouse clicks during gameplay'' doesn't work/doesn't bring back mouse clicks

### DISCUSSION THREAD

**Discussion by mandrake780** [ppy/osu#30854] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Wanted to play mouse only. Disabled ''Disable mouse clicks during gameplay'' but mouse clicks are still not working. The mouse itself works on other parts of Lazer, but not gameplay. I tried restarting the client and still had the same issue. I am on MacOS Seqouia (ARM).

### Screenshots or videos

https://github.com/user-attachments/assets/116ae7bc-d794-4296-a000-b6f7a5dfd602

### Version

2024.1115.3-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/17882708/compressed-logs.zip)

**Comment by frenzibyte** [comment, marked as answer]
Check back on your game's key binding mapping, you've probably disassociated mouse buttons from osu! actions.
